code/modules/client/preferences.dm

```diff
@@ -1063,8 +1061,6 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					yogtoggles ^= QUIET_ROUND
 				if("pda")
 					donor_pda = donor_pda % donor_pdas.len + 1
-				if("purrbation")
```

Comment:
This also removes catboys as well as catgirls. This is clearly an error

Reply:
no its not